src/backend/executor/nodeHash.c

```diff
@@ -768,11 +768,20 @@ ExecHashIncreaseNumBatches(HashJoinTable hashtable)
 	/* EXPLAIN ANALYZE batch statistics */
 	if (stats && stats->nbatchstats < nbatch)
 	{
-		Size		sz = nbatch * sizeof(stats->batchstats[0]);
+		Size		sz = nbatch * sizeof(HashJoinBatchStats);
```

Comment:
Is `sizeof(HashJoinBatchStats)` equal to `sizeof(stats->batchstats[0])` or not?

Reply:
Yes, it is

Reply:
then remove unnecessary changes

Reply:
Done. I just though this was slightly confusing, because everything else in this function uses types for `sizeof`.